not_so_basic_grunt.py

```diff
@@ -0,0 +1,100 @@
+import pygame
+import math
+
+from gameobject import GameObject
+
+
+class NotSoBasicGrunt(GameObject):
+    
+
+    def __init__(self, starting_position, universe):
+        self.position_ = starting_position
+
+        self.ID_ = self.create_ID()
+        self.size_ = 15
+        self.health_ = 30
+        self.universe_ = universe
+        self.listeners_ = []
+        self.enemy_type_ = "NotSoBasicGrunt"
+
+        self.x_velocity = 0
+        self.y_velocity = 0.3 
+        self.x_accel = 0
+        self.y_accel = 0
+
+        self.gravity_hole_1 = (500,300)
+
+    """
+    Update Functions
+    """
+
+    def update(self, events):
+        force_1 = self.calculate_force_from_(self.gravity_hole_1)
```

Comment:
You probably don't know about list comprehensions, but when you do implement this as lists, list comprehensions would be way to go. So this case would look something like:
`forces = [self.calculate_force_from(gravityhole) for gravityhole in gravityholes]